### PR TITLE
fixed ckan.ico path

### LINKED ISSUE #2900: Ckan Favicon with wrong path
### CKAN Version if known (or site URL)
2.5.1, from source

### Please describe the expected behaviour
Load ckan icon

### Please describe the actual behaviour
Can't lada ckan icon

### What steps can be taken to reproduce the issue? 
Open dev console on a browser. Quick search on github shows the path is wrong on the source code. it points to 'images/icons/ckan.ico' instead of 'base/images/icons/ckan.ico'

### PR DESCRIPTION
**Pull request by bunnis**
Fixes #2900 

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

File: ckan/config/deployment.ini_tmpl [hunk 1]
```diff
@@ -107,7 +107,7 @@ ckan.views.default_views = image_view text_view recline_view
 ckan.site_title = CKAN
 ckan.site_logo = /base/images/ckan-logo.png
 ckan.site_description =
-ckan.favicon = /images/icons/ckan.ico
+ckan.favicon = /base/images/icons/ckan.ico
```
Comment (amercader) [inline ckan/config/deployment.ini_tmpl]: Thanks for this @bunnis . The actual correct path is `/base/images/ckan.ico`:

https://github.com/ckan/ckan/blob/master/ckan/public/base/images/ckan.ico

Can you amend your PR so I can merge it?

File: ckan/config/environment.py [hunk 2]
```diff
@@ -246,7 +246,7 @@ def update_config():
         config['ckan.site_id'] = ckan_host
 
     # ensure that a favicon has been set
-    favicon = config.get('ckan.favicon', '/images/icons/ckan.ico')
+    favicon = config.get('ckan.favicon', '/base/images/icons/ckan.ico')
```
Comment (amercader) [inline ckan/config/environment.py]: Same here

**Comment by bunnis** [comment]
Actually you are right, but somehow I have the icon appearing on my ckan site. And I don't have a /base/images/icons/ folder. Is there a possibility the path is beeing changed somewhere else?

**Comment by bunnis** [comment]
I tripled checked my config and it also points to ckan.favicon = /base/images/icons/ckan.ico

the location of my ckan.ico
bunny@bunny-VirtualBox:~$ locate ckan.ico
/home/bunny/ckan/lib/default/src/ckan/ckan/public/base/images/ckan.ico

**Comment by amercader** [comment]
Some things to check off the top of my head:
* Duplicated `ckan.favicon` value on your ini file
* system_info table on the database
* Browser cache

In any case the proposed solution (`/base/images/ckan.ico`) is the good one to go.

**Comment by bunnis** [comment]
I checked browser cache and duplicate ckan.favicon, but I didn't check system_info table on the database. I still don't believe its from that since the icon only started to show after I changed the mentioned classes on this PR. Still I'm writing if a future reference on this is needed. I'll update the PR as you suggested since its the correct one.

**Comment by amercader** [comment]
Thanks for the patch @bunnis

**Comment by drnextgis** [comment]
When this changes will accessible in CKAN release?

**Pull request merged**